public/src/components/calls/callcntrl_btn_group.jsx

```diff
@@ -1,29 +1,21 @@
 import React from 'react';
-import { ButtonGroup, Button } from 'react-bootstrap';
+import { Button } from 'react-bootstrap';
 
 const CallControl = (props) => {
-  // TODO: SUBMIT DOES NOTHING RN... NOTHING
-  const { handleHangUp, submitHandler, outcome, status } = props;
+  const { handler, text, outcome, status, htmlID } = props;
+  const style = text === 'Hang Up' ? 'danger' : 'success';
```

Comment:
One could argue that we should do something similar to buttons where the component just accepts props for everything. But you know what. This PR has had enough review. We can let this slide. Progress over perfection.